modules/bazel-diff/7.2.13/MODULE.bazel

```diff
@@ -0,0 +1,32 @@
+module(
+    name = "bazel-diff",
+    version = "7.2.13",
+    compatibility_level = 0,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.6.1")
+bazel_dep(name = "rules_proto", version = "6.0.0-rc2")
+bazel_dep(name = "rules_java", version = "7.6.5")
+bazel_dep(name = "rules_kotlin", version = "1.9.1")
+bazel_dep(name = "rules_jvm_external", version = "6.0")
+
+bazel_diff_maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+bazel_diff_maven.install(
```

Comment:
this is not what @fmeum meant with a different name. What you call the variable is irrelevant -- you can call it `foo` and nothing would change. What you need to do is to explicitly pass `name = "bazel_diff_maven"` to the `install` tag, like this: https://github.com/bazelbuild/bazel-central-registry/blob/aadc90c53943809be4b6b93793acf82ab5877094/modules/copybara/0.0.0-20240326-6485af6/MODULE.bazel#L23